src/suffer.cpp

```diff
@@ -794,16 +783,73 @@ void Character::suffer_in_sunlight()
     }
 }
 
-void Character::suffer_from_albinism()
+std::map<bodypart_id, float> Character::bodypart_exposure()
+{
+    std::map<bodypart_id, float> bp_exposure;
+
+    // Set of all body parts affected by clothing coverage
+    // FIXME: Factor this out (same list is used in player.cpp)
+    body_part_set affected_bp { {
```

Comment:
could this be move the json definition of the mutation? There might already be an affected body part entry there.

Reply:
I do not see any fields in mutations that could be used for affected body parts - we have numerous other fields that include body parts like "lumination", "no_cbm_on_bp", "armor", "encumbrance_covered", "wet_protection", and "restricts_gear", but it's not clear how I might repurpose those for this, and I'm hesitant to add even more fields, at least not in this PR.

Reply:
You're right, maybe I meant to add that at some point and then didn't... U_U" Disregard this suggestion then ^^"

Reply:
Shouldn't this one *also* just use `get_all_body_parts`?  Is it different from the list of all body parts?

Reply:
Sure enough, this list has all the same parts as `get_all_body_parts` now (it used to exclude the eyes, but not anymore). So this function became somewhat simpler in 146ef9d81